custom/icds_reports/utils/fix_past_data_due_to_change_in_height_eligibilty.sql

```diff
@@ -0,0 +1,358 @@
+-- child health table fix
```

Comment:
queries in this file looks duplicate to what I see above. Is this file here fore some special reason?

Reply:
my bad it's the code from some previous similar PR